rules/validate-kubelet-tls-configuration-updated/raw.rego

```diff
@@ -0,0 +1,54 @@
+package armo_builtins
+
+deny[msga] {
+	kubelet_info := input[_]
+	kubelet_info.kind == "KubeletInfo"
+	kubelet_info.apiVersion == "hostdata.kubescape.cloud/v1beta0"
+
+	command := kubelet_info.data.cmdLine
+
+	not contains(command, "--tls-cert-file")
+	not contains(command, "--tls-private-key-file")
+	not contains(command, "--config")
```

Comment:
What if let's say `--tls-cert-file` is set but not `--tls-cert-file`? It still suppose to fall, but this won't happen

Reply:
the rule has been completely refactored, your case is handled as well as combined assignment via config and cli